test/file.rb

```diff
@@ -76,8 +80,13 @@
 end
 
 assert('File.realpath') do
-  usrbin = IO.popen("cd bin; /bin/pwd -P") { |f| f.read.chomp }
-  assert_equal usrbin, File.realpath("bin")
+  if File.const_defined?(:ALT_SEPARATOR) && File::ALT_SEPARATOR
```

Comment:
You're right.  I have used `const_defined?` before #55 was merged.  But now we can remove it.